docs/development/building.md

```diff
@@ -47,5 +47,5 @@ and then configure your IDE to connect its debugger to port 2345 on localhost.
 
  - Make sure `$GOPATH` is set, and your [workspace](https://golang.org/doc/code.html#Workspaces) is configured.
  - kops will not compile with symlinks in `$GOPATH`. See issue go issue [17451](https://github.com/golang/go/issues/17451) for more information
- - building kops requires go 1.12 or 1.13
+ - building kops requires go 1.14 or 1.15
```

Comment:
That is quite a good change. Always forget to change here. Could we say just that build requires 1.15?
Would avoid confusions when some generate / update / test command fails because of 1.14. We don't test anymore with 1.14 at all.
```suggestion
 - building kops requires go 1.15
```

Reply:
Works for me. Although, I didn't experience any build issues with version `go1.14`, I'm always happy to go with the latest version!

Reply:
Thanks. There are a few `hack/verify-*` and `hack/update-*` commands that don't work with 1.14. Better not confuse people.